src/components/menu.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * @file Contains an interactive drawermenu
+ */
+import React, { useState } from 'react';
+import { styled, useTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Drawer from '@mui/material/Drawer';
+import CssBaseline from '@mui/material/CssBaseline';
+import MuiAppBar, { AppBarProps as MuiAppBarProps } from '@mui/material/AppBar';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import Example from './example';
+import DragAndDropzone from './DragAndDropzone';
+import { GraphComponent, InfoboxComponent } from './BaseComponent';
```

Comment:
import breaks 
![image](https://user-images.githubusercontent.com/95719901/229809867-8ce4a7c6-496a-470f-88d3-5245d4deadd9.png)

should be lowercase